### PR TITLE
Added publisher for ground truth camera_localized_pose

### PR DESCRIPTION
**Pull request by amiller27**

### CODE REVIEW

File: src/simulator_adapter.py [hunk 1]
```diff
@@ -50,6 +50,16 @@ def sim_pose_callback(pose_msg):
 
         tf2_broadcaster.sendTransform(transform_msg)
 
+    if publish_ground_truth_camera_localization:
+        camera_pose_msg = PoseWithCovarianceStamped()
+        camera_pose_msg.header.stamp = pose_msg.header.stamp
+        camera_pose_msg.header.frame_id = 'map'
+        camera_pose_msg.pose.pose.position = pose_msg.pose.position
```
Comment (aftersomemath) [inline src/simulator_adapter.py:57]: Should we just copy the parts of the pose out that the camera would actually report?

**Review by aftersomemath — APPROVED** [review]

File: src/simulator_adapter.py (hunk 1 above)
Comment (amiller27) [reply src/simulator_adapter.py:57]: That's what I'm doing, the camera measures all 3 position axes

Comment (amiller27) [reply src/simulator_adapter.py:57]: I'm throwing out the rotation information

Comment (aftersomemath) [reply src/simulator_adapter.py:57]: Ah I see very good

**Pull request merged**